athena-federation-sdk/src/main/java/com/amazonaws/athena/connector/lambda/domain/predicate/Constraints.java

```diff
@@ -58,6 +76,26 @@ public Map<String, ValueSet> getSummary()
         return summary;
     }
 
+    public List<FederationExpression> getExpression()
+    {
+        return expression;
+    }
+
+    public long getLimit()
+    {
+        return limit;
+    }
+
+    public boolean hasLimit()
+    {
+        return this.limit > -1;
```

Comment:
lets use the new static variable suggested earlier here as well.

Reply:
good idea, just pushed this change